app/mailers/notifications_mailer.rb

```diff
@@ -4,7 +4,19 @@ class NotificationsMailer < ActionMailer::Base
 
   def new_notification(notification_id)
     @notification = Notification.where(id: notification_id).first
+    @discussion = @notification.discussion
+    @url = url_for_notification(@notification)
 
-    mail to: @notification.email, subject: "[Council] #{@notification.text}"
+    mail to: @notification.email, subject: '[Council] New discussion', template_name: @notification.kind
+  end
+
+  private
+
+  def url_for_notification(notification)
+    case notification.kind
+    when 'new_discussion' then root_url(anchor: "/discussions/#{notification.discussion.id}")
+    when 'new_comment'    then root_url(anchor: "/discussions/#{notification.discussion.id}/comments/#{notification.discussion.comments.last.id}")
```

Comment:
Can't you make this a little nicer? Maybe wrap the `anchor: "/discussions/#{notification.discussion.id}"` in private methods and use the return value of `case` as argument to the `root_url`.